app/assets/stylesheets/frontend/print/_html-publication.scss

```diff
@@ -53,4 +53,20 @@
       padding: 0;
     }
   }
+
+  #meta-data {
+    page-break-before: always;
+    page-break-after: always;
+
+    #meta-data * {
```

Comment:
Does this selector actually do anything?  I think scss will turn this into `#meta-data #meta-data *` which won't match, so I suspect we can drop this entirely.